joplin.desktop

```diff
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Version=1.0.178
```

Comment:
Why do you want to remove the version from the `desktop`file ?

Reply:
A comment on the AUR claimed that this is actually the version of the freedesktop spec used, and is completely optional. I checked and the spec seemed to confirm this.